libvirt.go

```diff
@@ -521,6 +521,11 @@ func (d *Driver) getIPByMacFromSettings(mac string) (string, error) {
 	}
```

Comment:
Regarding the commit log, what happens if we don't ignore empty files? I assume the `log.Warnf("Failed to decode dnsmasq lease status: %s", err)` block triggers and we return an error? Can you mention it in the commit log?